src/r-packages/devcontainer-feature.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "R packages (via pak)",
+	"id": "r-packages",
+	"version": "1.0.0",
+	"description": "Installs R packages via the pak R package's function.",
+	"documentationURL": "https://github.com/rocker-org/devcontainer-features/tree/main/src/r-packages",
+	"options": {
+		"packages": {
+			"type": "string",
+			"default": "",
+			"description": "Comma separated list of packages to install to pass to the `pak::pak()` function."
+		},
+		"pakVersion": {
+			"type": "string",
+			"enum": [
+				"auto",
+				"devel",
+				"stable"
+			],
+			"default": "auto",
+			"description": "Version of pak to install. By default, the devel version is installed if needed."
+		},
+		"additionalRepositories": {
+			"type": "string",
+			"default": "",
+			"description": "String passed to the `pak::repo_add()` function."
+		}
+	},
+	"installsAfter": [
+		"ghcr.io/devcontainers/features/common-utils",
+		"ghcr.io/rocker-org/devcontainer-features/apt-packages",
+		"ghcr.io/rocker-org/devcontainer-features/r-apt",
+		"ghcr.io/rocker-org/devcontainer-features/r-rig"
+	]
```

Comment:
It seems that `installsAfter` is not being respected during the test and the test is failing.
I created an issue. devcontainers/cli#347